doc/modules/reference.rst

```diff
@@ -115,7 +137,8 @@ uses.
    reorder_img
    smooth_img
    swap_img_hemispheres
-
+   coord_transform
+   get_bounds
```

Comment:
Ok, what's the nilearn policy for exposing / not-exposing functions ? I
guessed we'd want to expose stuff by default, otherwise we'd make them
"hidden" by prepending the "_" marker, but i may be wrong.

On Sat, Sep 26, 2015 at 10:42 AM, Gael Varoquaux <notifications@github.com>
wrote:

> In doc/modules/reference.rst
> <https://github.com/nilearn/nilearn/pull/780#discussion_r40491694>:
>
> > @@ -115,7 +137,8 @@ uses.
> >     reorder_img
> >     smooth_img
> >     swap_img_hemispheres
> > -
> > +   coord_transform
> > +   get_bounds
>
> I don't think that we want to expose this function: to me, it's internal.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/780/files#r40491694>.
>



-- 
DED